.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
(BTW the [`type`](https://getcomposer.org/doc/04-schema.md#type) is currently missing from the [`composer.json` file](https://github.com/manala/manalize/blob/master/composer.json) ^^'